src/main/flight/pid.c

```diff
@@ -515,8 +515,14 @@ static void detectAndSetCrashRecovery(
     }
 }
 
-static void handleItermRotation(const float deltaT)
+static void handleItermRotation(const timeUs_t currentTimeUs)
```

Comment:
Just moving jitter into I term rotation (where it is quite invisible)

Reply:
Yes, this is what I wanted to discuss. Would be good to change it also? ItermRotation is already working with deltaT and so far no one complained.

If we agree that it should be redone also, I will be more than happy to do it :)

Reply:
![image](https://user-images.githubusercontent.com/28888/39047087-ef623c64-4498-11e8-910f-98e4eb6ff839.png)
I-Term seem to be unaffected by jitter so far (red line)

Reply:
Problem is that sampling timing is internal to gyro(and with negligible jitter). Unless gyro-loop jitter is larger than gyro sampling period (loosing some gyro samples), constant dT is almost exact.

Reply:
So it means that you are voting for constant dT in ItermRotation also? If so, I can do it if there will be no objections :)